frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/StaticEmbedSetupPane.styled.tsx

```diff
@@ -0,0 +1,89 @@
+import styled from "@emotion/styled";
+import type { ReactNode } from "react";
+import { color } from "metabase/lib/colors";
+import { Box, Center, Text } from "metabase/ui";
+import { Icon } from "metabase/core/components/Icon";
+
+const ContentWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: stretch;
+
+  min-height: 648px;
+`;
+
+const SettingsAsideBlock = styled.div`
+  flex-shrink: 0;
+  width: 345px;
+  padding: 2rem;
+  border-right: 1px solid ${color("border")};
+  background-color: ${color("white")};
+`;
+
+const PreviewAreaBlock = styled.div`
+  width: 100%;
+  min-width: 810px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: stretch;
+
+  gap: 1rem;
+
+  padding: 0.5rem 1.5rem 2rem;
+  background-color: ${color("bg-light")};
+`;
+
+export const SettingsTabLayout = ({
+  settingsSlot,
+  previewSlot,
+}: {
+  settingsSlot: ReactNode;
+  previewSlot: ReactNode;
+}): JSX.Element => {
+  return (
+    <ContentWrapper>
+      <SettingsAsideBlock>{settingsSlot}</SettingsAsideBlock>
+      <PreviewAreaBlock>{previewSlot}</PreviewAreaBlock>
+    </ContentWrapper>
+  );
+};
+
+export const CODE_PREVIEW_CONTROL_OPTIONS = [
```

Comment:
This doesn't feel like it belongs in this file as it doesn't have anything to do with styling. Am I correct?